package.json

```diff
@@ -5,7 +5,7 @@
   "main": "lib/sumoLogger.js",
   "scripts": {
     "release": "np",
-    "test": "nyc mocha --exit",
+    "test": "nyc jest",
     "cover": "nyc --reporter=lcov npm test",
```

Comment:
I guess some changes will need to be done here to enable coverage?

Reply:
I'm actually thinking of removing the coverage altogether, because I don't believe it makes much sense for such a small repo.